jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -378,7 +378,9 @@ private static void ensureProperties() {
 	systemProperties.put("sun.jnu.encoding", platformEncoding); //$NON-NLS-1$
 	systemProperties.put("file.encoding", fileEncoding); //$NON-NLS-1$
 	systemProperties.put("file.encoding.pkg", "sun.io"); //$NON-NLS-1$ //$NON-NLS-2$
+	/*[IF !Java12]*/
 	systemProperties.put("java.specification.vendor", "Oracle Corporation"); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
I assume java.specification.vendor is set by VersionProps.init()? Please add a comment.

Reply:
That's right. Added a comment.
Ready for another review.